src/App.js

```diff
@@ -4,12 +4,23 @@ function App() {
   return (
     <div className="App">
       <header className="App-header">My cakes</header>
-      <main className="App-main"></main>
+      <main className="App-main">
```

Comment:
looks good! i only would remove from the classname the word "App". i would just leave it main...

Reply:
It makes sense to remove it indeed. Thx !